CMakeLists.txt

```
@@ -4,5 +4,7 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 include_directories("inipp/")
 install(FILES inipp/inipp.h DESTINATION include)
-enable_testing()
-add_subdirectory("unittest/")
+if (BUILD_TESTING)
+    enable_testing()
```

BUILD_TESTING is only set when CTest is included. Also CTest will call enable_testing. Does this work for you?

```suggestion
include(CTest)
if(BUILD_TESTING)
```

---

Oups, it seems like we both did the same thing at the same time

---

LOL yes 😄